<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>980</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>11</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>New Purchases</string>
  </property>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>1790</x>
     <y>40</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>News706 BT</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Images/20.png</normaloff>../Images/20.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Purchase Order Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Particulars</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Order Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Supplier Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>130</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Contact Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Item Name</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>60</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>60</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>130</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Place of Supply</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>270</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Unit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>270</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>1085</x>
     <y>270</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Purchase Price</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>1335</x>
     <y>270</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>GST %</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>1580</x>
     <y>270</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Final Amount</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="shipping_CheckBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>740</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Shipping Cost</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>790</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Payment Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>790</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Transaction ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>880</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount Paid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>880</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Balance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>750</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Sub Total</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>820</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>730</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Payments</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>380</width>
     <height>3</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>3</width>
     <height>160</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>30</y>
     <width>1200</width>
     <height>3</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>1870</x>
     <y>30</y>
     <width>3</width>
     <height>160</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="orderType_CB">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>NON-GST</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GST</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="supplier_CB">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>1850</width>
     <height>3</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="contact_Edit">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>130</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="place_Edit">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>130</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>380</width>
     <height>3</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>3</width>
     <height>470</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>1370</width>
     <height>3</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>1870</x>
     <y>240</y>
     <width>3</width>
     <height>470</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="itemName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addItemButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>112</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="unit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>320</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="quantity">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>320</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="purchasePrice">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>320</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>320</y>
     <width>30</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Rs.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gst">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>320</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="finalAmount">
   <property name="geometry">
    <rect>
     <x>1570</x>
     <y>320</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>320</y>
     <width>30</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Rs.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteItem">
   <property name="geometry">
    <rect>
     <x>1810</x>
     <y>340</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>1800</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Sr. No.</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Item Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Unit</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Purchase Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>GST %</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Final Amount</string>
    </property>
   </column>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>710</y>
     <width>1850</width>
     <height>3</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="shipAmount_Edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>810</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>810</y>
     <width>30</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Rs.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reminderButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>900</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Reminder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>900</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>1710</x>
     <y>750</y>
     <width>30</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Rs.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>1710</x>
     <y>820</y>
     <width>30</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Rs.</string>
   </property>
  </widget>
  <widget class="QLabel" name="subAmount_label">
   <property name="geometry">
    <rect>
     <x>1750</x>
     <y>750</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string></string>
   </property>
  </widget>
  <widget class="QLabel" name="tamount_label">
   <property name="geometry">
    <rect>
     <x>1750</x>
     <y>820</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string></string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>880</y>
     <width>30</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Rs.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amountPaid_Edit">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>880</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>880</y>
     <width>30</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Rs.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="balance_Edit">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>880</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="paymode_CB">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>790</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Cash</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cheque</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Online</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bank Transfer</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="transactionId_Edit">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>790</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>1470</x>
     <y>750</y>
     <width>3</width>
     <height>200</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_12">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>750</y>
     <width>3</width>
     <height>200</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_13">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>750</y>
     <width>380</width>
     <height>3</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_14">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>750</y>
     <width>300</width>
     <height>3</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_15">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>950</y>
     <width>790</width>
     <height>3</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>740</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Remarks</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="remarksTextEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>810</y>
     <width>300</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>1670</x>
     <y>50</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>60</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Purchase ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="purchaseId">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>60</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addItem">
   <property name="geometry">
    <rect>
     <x>1760</x>
     <y>340</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addSupplier">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>130</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
